lib/monitor.js

```diff
@@ -52,14 +52,13 @@ module.exports = internals.Monitor = class {
 
             this.push(() => new Utils.Ops(results));
         };
-<<<<<<< HEAD
 
         this.configure(options);
     }
 
     configure(options) {
 
-        if (this._reporters.size > 0 || this._extensionListeners.length > 0) {
+        if (this._requiresStop) {
```

Comment:
This line caused a coverage issue because we weren't testing both conditions, so I switched instead to a stop-specific flag.